guides/micronaut-oauth2-cognito/micronaut-oauth2-cognito.adoc

```diff
@@ -53,7 +53,7 @@ export COGNITO_POOL_ID=eu-west-XXCCCAZZZ
 
 Although undocumented in the https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-userpools-server-contract-reference.html[Amazon Cognito User Pools Auth API Reference], Cognito provides an https://openid.net/specs/openid-connect-discovery-1_0.html[openid-configuration endpoint] which the Micronaut framework uses to configure your application.
 
-[source, bash]
+[source, bash,subs="attributes+"]
```

Comment:
why ?

Reply:
it's needed when we will use asciidoc to include the file instead of manually doing it (at the moment though, it has no effect)

Reply:
@AndreaLaGrotteria Since it is not doing nothing currently, leave it as it was.